vllm/distributed/kv_transfer/kv_connector/v1/nixl_connector.py

```diff
@@ -296,12 +318,16 @@ def __init__(self, vllm_config: VllmConfig, engine_id: str):
         self.side_channel_port = (
             envs.VLLM_NIXL_SIDE_CHANNEL_PORT
```

Comment:
note to readers: we go from DP*TP open ports here to DP ports

Reply:
@GuanLuo would you mind updating nixl connector usage docs where it mentions the open ports 
`For TP/DP deployments, each worker's port on...`